Sources/NIOHTTP2/HTTP2Error.swift

```diff
@@ -344,29 +347,72 @@ public enum NIOHTTP2Errors {
             }
         }
 
-        private final class Storage: Equatable {
-            var delta: Int
-            var currentWindowSize: Int
-            var file: String
-            var line: UInt
+        fileprivate final class Storage: Equatable {
```

Comment:
We don't need this: the backing storage is only accessed after a call to `self.copyStorageIfNotUniquelyReferenced`, so we will never write to this across threads. This is the perfect case for making the backing storage _not_ `Sendable`, and instead having the wrapper `struct` be `@unchecked Sendable`.